reporting/accuracy/plot_jacobi_theta1.cpp

```diff
@@ -0,0 +1,31 @@
+
+#include <iostream>
+#include <boost/math/tools/ulps_plot.hpp>
+#include <boost/core/demangle.hpp>
+#include <boost/math/special_functions/jacobi_theta.hpp>
+
+using boost::math::tools::ulps_plot;
+
+int main() {
+    using PreciseReal = long double; // boost::multiprecision::float128;
+    using CoarseReal = double;
```

Comment:
In this case, there is no problem, but in general I like to use `float128` to test `double`, or use `long double` to test `float`. Obviously that isn't sufficient to guarantee accuracy of the plot, but I have had problems where `long double` wasn't sufficient to get a good ulps plot for `double`.

Reply:
`boost::multiprecision::float128` does not compile on my machine - that's why I switched it to `long double`.

Reply:
Ok, then I'd use `using CoarseReal = float`; it should only depend on the difference in precisions.